app/models/subreddit.rb

```diff
@@ -0,0 +1,26 @@
+class Subreddit
+
+  attr_accessor :rules
+
+  attr_reader :name
+
+  def initialize(name, token)
+    @name = name
+    @rules = Array.new
+    @token = token
+  end
+
+  def get_rules
+    serv = RedditService.new(@token)
```

Comment:
I know that there is a way to refactor this, but I'm not entirely sure how. I also feel like storing the token directly on the subreddit PORO is not ideal. Any ideas on how to refactor? @s-espinosa, @AliSchlereth